src/commonMain/kotlin/co/remotectrl/ctrl/event/CtrlAggregateCommandResult.kt

```diff
@@ -0,0 +1,6 @@
+package co.remotectrl.ctrl.event
+
+data class CtrlAggregateCommandResult<TMutable : CtrlMutable<TMutable>>(
```

Comment:
If this is to be public facing, I'd add a few doc strings with `@see` and an example code block

Reply:
Want people to refer to tests initially. If I document would like to start with vue-press